.github/workflows/cd.yml

```diff
@@ -37,12 +40,12 @@ jobs:
               run: yarn ci
 
             - name: Set Git user name
-              run: git config --global user.name "Fernando Hurtado"
+              run: git config user.name "github-actions[bot]"
 
             - name: Set Git user email
-              run: git config --global user.email "fernandohur@gmail.com"
+              run: git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where is this ID/email coming from? Can you add a bit of documentation to the yaml file explaining?